pimegaApp/Db/module.template

```diff
@@ -0,0 +1,104 @@
+#=================================================================#
+# Template file: module.template
+# Database for PIMEGA module-based records
+
+record(waveform, "$(P)$(R)MB_Temperature_RBV") {
+   field(DESC, "All temperatures on MB sensors")
+   field(NELM, "48")
+   field(FTVL, "FLOAT")
+   field(DTYP, "asynFloat32ArrayIn")
+   field(INP,  "@asyn($(PORT),$(ADDR),$(TIMEOUT))MODULE_MB_TEMPERATURE")
+   field(SCAN, "I/O Intr")
+   field(PREC, "2")
+}
+
+record(ai,"$(P)$(R)MB_AvgTemperature_RBV") {
+   field(DESC, "MFB average temperature")
+   field(DTYP, "asynFloat64")
+   field(INP,  "@asyn($(PORT),$(ADDR),$(TIMEOUT))MODULE_MB_AVG_TSENSOR")
+   field(SCAN, "I/O Intr")
+   field(PREC, "2")
+   field(EGU, "°C")
+}
+
+record(waveform, "$(P)$(R)Sensor_Temperature_RBV") {
+   field(DESC, "All sensors temperatures")
+   field(NELM, "36")
+   field(FTVL, "FLOAT")
+   field(DTYP, "asynFloat32ArrayIn")
+   field(INP,  "@asyn($(PORT),$(ADDR),$(TIMEOUT))MODULE_SENSOR_TEMPERATURE")
+   field(SCAN, "I/O Intr")
+   field(PREC, "2")
+}
+
+record(ai,"$(P)$(R)Medipix_AvgTemperature_RBV") {
+   field(DESC, "Medipix average temperature")
+   field(DTYP, "asynFloat64")
+   field(INP,  "@asyn($(PORT),$(ADDR),$(TIMEOUT))MODULE_MP_AVG_TSENSOR")
+   field(SCAN, "I/O Intr")
+   field(PREC, "2")
+   field(EGU, "°C")
+}
+
+#each status controls a bit, thus using values 0, 1, 2, 4
+record(mbbi,"$(P)$(R)Temperature_Status") {
+   field(DESC, "Medipix board temperature status")
+   field(DTYP, "asynInt32")
+   field(INP,  "@asyn($(PORT),$(ADDR),$(TIMEOUT))MODULE_TEMP_STATUS")
+   field(ZRST, "No Support")
+   field(ZRVL, "0")
+   field(ONST, "Normal")
+   field(ONVL, "1")
+   field(TWST, "Warning")
+   field(TWVL, "2")
+   field(FRST, "Critical")
+   field(FRVL, "4")
+   field(PREC, "0")
```

Comment:
I think we can take the opportunity to clean up `PREC` from here.

Reply:
Done in along with "Move module-specific records to dedicated template".